Procfile

```diff
@@ -10,6 +10,6 @@
 #
 # Alternatively, these values can be set as normal environment variables.
 
-api: cd api && node server.js
+api: grunt dev-api
 webapp: grunt dev-no-npm
```

Comment:
`grunt dev-webapp`?

Reply:
Was thinking about that but didn't want to rock the boat, will happily update

Reply:
and `grunt dev` might be better called `grunt dev-webapp-with-npm`.  But you might want to run this past @garethbowen first.  Maybe get this PR merged, and then discuss?

Reply:
`dev-webapp-with-npm` is a bit long for me. What I really want is a way to run `npm install` only when the package.json has changed (new dep or whatever) because it's a PITA to get failures for this. Maybe we need a custom script which hashes/caches the `package.json` and runs `npm install` only when it's changed. I think that would mean everyone could use `grunt dev` and we wouldn't have to have long names and multiple variants for different performance profiles.